src/ActivitiesSDKAdapter.js

```diff
@@ -168,4 +136,64 @@ export default class ActivitiesSDKAdapter extends ActivitiesAdapter {
 
     return activity$;
   }
+
+  /**
+   * A function that checks whether or not an Activity object contains a card attachment.
+   *
+   * @param {Activity} activity  Activity object
+   * @returns {boolean} True if received Activity object contains a card attachment
+   */
+  // eslint-disable-next-line class-methods-use-this
+  hasAdaptiveCard(activity) {
+    return !!activity.attachments && activity.attachments[0] && activity.attachments[0].contentType === 'application/vnd.microsoft.card.adaptive';
+  }
+
+  /**
+   * A function that returns adaptive card data of an Activity object.
+   *
+   * @param {Activity} activity  Activity object
+   * @returns {object} Adaptive card data object
+   */
+  // eslint-disable-next-line class-methods-use-this
+  getAdaptiveCard(activity) {
+    const hasCard = this.hasAdaptiveCard(activity);
+
+    return hasCard ? activity.attachments[0].content : undefined;
+  }
+
+  /**
+   * A function that attaches an adaptive card to an Activity object.
+   *
+   * @param {Activity} activity  The activity to post
+   * @param {object} card  The card attachment
+   */
+  // eslint-disable-next-line class-methods-use-this
+  attachAdaptiveCard(activity, card) {
+    const mutableActivity = activity;
+
+    mutableActivity.attachments = [{
+      contenType: 'application/vnd.microsoft.card.adaptive',
+      content: card,
+    }];
+  }
+
+  /**
+   * Maps SDK activity to adapter activity
+   *
+   * @param {object} sdkActivity  SDK activity object
+   * @returns {Activity} Adapter activity object
+   */
+  // eslint-disable-next-line class-methods-use-this
+  fromSDKActivity(sdkActivity) {
```

Comment:
you can move this function outside the class, just like `getMembers` in memberships adapter:

```js
/**
 * Maps SDK members to adapter members
 *
 * @private
 * @param {object} sdkMembers  Members object from sdk meeting, keyed by ID
 * @param {object} meeting  Sdk meeting object
 * @returns {Array.<Member>} List of meeting members
 */
function getMembers(sdkMembers, meeting) {